### ISSUE TITLE
Grapher multiple columns plot

### ISSUE THREAD

**Issue by adamerose** [adamerose/PandasGUI#192]
<div type='discussions-op-text'>

<sup>Originally posted by **gemGreg** January 31, 2022</sup>
In the old version of PandasGUI there was a way to plot multiple signals on one line plot.
![image](https://user-images.githubusercontent.com/7453447/151777966-afad982e-08a3-4a62-b623-edb8456558ce.png)
Is there a way to do it in the newest version of PandasGUI? 
It seems like In the newest version you can drag/drop only one signal to X and Y:
![image](https://user-images.githubusercontent.com/7453447/151778169-80e10534-58e8-43b4-b1d7-af4bdf22e6cd.png)
</div>

**Comment by blixtdevelopment** [comment]
Yep I have the same problem on version 0.2.13. Seems like an important feature to be able to plot multiple columns in the same plot.

**Comment by gemGreg** [comment]
do we have any update on when this issue could be fixed?

**Comment by michaelwills** [comment]
One workaround available now at least via the repo is to use custom kwargs to override `y` with an array of columns. It isn't as easy perhaps but doable. So in @adamerose 's case, add `y` with a value of `['Australia New Cases', 'Brunei New Cases', ...]`.

**Comment by peace712** [comment]
Below is what I think a better workaround until this gets fixed.
My use case for PandasGUI is almost exclusively comparing and analyzing time series data. Being able to plot multiple columns is a "must have" feature, at least for line and scatter plot types.  I was already making some changes to read couple more file formats, and made the below changes for the line and scatter plot types to get that feature back. 

file -> jotly.py
function -> line and scatter
original argument-> `y: ColumnName = None,`
changed argument -> `y: ColumnNameList= None,`

Essentially, it changes the class of argument `y` from `ColumnName` to `ColumnNameList`
